bumpversion/cli.py

```diff
@@ -127,7 +127,7 @@ def main(original_args=None):
 
     # store the new version
     _update_config_file(
-        config, config_file, config_newlines, config_file_exists, args.new_version, args.dry_run,
+        config, config_file, config_newlines, config_file_exists, args.new_version, args.dry_run, args.no_update_config,
```

Comment:
Can we bypass `_update_config_file` altogether in case of `no_update_config` ?

Reply:
Could do that too, would you prefer that? Was trying to avoid flow changes as much as possible.